token/core/cmd/pp/common/common.go

```diff
@@ -68,3 +86,60 @@ func SetupIssuersAndAuditors(pp PP, Auditors, Issuers []string) error {
 	}
 	return nil
 }
+
+func ReadFile(file string) ([]byte, error) {
+	fileCont, err := ioutil.ReadFile(file)
+	if err != nil {
+		return nil, errors.Wrapf(err, "could not read file %s", file)
+	}
+
+	return fileCont, nil
+}
+
+func ReadPemFile(file string) ([]byte, error) {
+	bytes, err := ReadFile(file)
+	if err != nil {
+		return nil, errors.Wrapf(err, "reading from file %s failed", file)
+	}
+
+	b, rest := pem.Decode(bytes)
+	if b == nil {
+		return nil, errors.Errorf("no pem content for file %s", file)
+	}
+	if len(rest) != 0 {
+		return nil, errors.Errorf("extra content after pem file %s", file)
+	}
+	if b.Type != "CERTIFICATE" {
+		return nil, errors.Errorf("pem file %s is not a certificate", file)
+	}
+
+	return bytes, nil
```

Comment:
should we just return b?

Reply:
if we force only one cert, we can return directly the bytes